etc/adminhtml/menu.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0"?>
+
+<config xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:module:Magento_Backend:etc/menu.xsd">
+    <menu>
+        <add id="Inchoo_ProductBookmark::productbookmark"
```

Comment:
šta ovo radi pod contentom u adminu? jel nije customers tab ili products nekako prikladniji za to (customers ako prikazuje nešto vezano za customere, a products ako pokazuje samo statistku vezanu za produkte koliko su puta bookmarkani kao u tvom slučaju)?

Reply:
Prebaceno